telemetry/definitions/commonDefinitions.json

```diff
@@ -277,6 +277,16 @@
             "type": "string",
             "description": "The tool being installed",
             "allowedValues": ["session-manager-plugin", "dotnet-lambda-deploy", "dotnet-deploy-cli"]
+        },
+        {
+            "name": "successCount",
+            "type": "double",
+            "description": "The number of successful operations"
+        },
+        {
+            "name": "failedCount",
+            "type": "double",
```

Comment:
is there a reason why we want this to be double? since it's a count I believe it should be fine being an int.

Reply:
initial rationale was to prevent any chances of overflow. Changed to int since that's not much of an issue